server/models.py

```diff
@@ -0,0 +1,48 @@
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
+
+db = SQLAlchemy()
+
+# Timezone should default to the meeting creators timezone?
+class Appointment(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    meeting_id = db.Column(db.Integer, db.ForeignKey("meeting.id"))
+    name = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(80), unique=False, nullable=False)
+    time = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    timezone = db.Column(db.String(50), nullable=False)
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    updated_at = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+
+
+class Meeting(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey("user.id"))
+    name = db.Column(db.String(255), nullable=False, default="60 Minute Meeting")
+    description = db.Column(db.Text, default="One-on-one")
+    duration = db.Column(db.Integer, nullable=False, default=60)
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    updated_at = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    appointments = db.relationship("Appointment", backref="meeting")
+
+
+class Availability(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey("user.id"))
+    day = db.Column(db.Integer, nullable=False)
```

Comment:
This may have to be a Date

Reply:
I was thinking about storing the day of the week, so rather than DateTime it would be 0 for Monday, 1 for Tuesday, 2 for Wednesday, etc. Does this design sound alright?

Reply:
Probably better to be a date - if the user wanted to be able to choose from 2 weeks of availability it would come in handy.

Reply:
So, if that is the case, the availability would be updated every week by the user or whenever they like? From my understanding, you are saying if the user wanted to put in two separate Mondays with different availability and duration, the date design would be better correct?
- If that is the case, the mock-ups didn't show it so is this just for the purpose of better design? (or maybe I missed it on the mockup)
- If that is the case, to make it more modular and more flexible, should the start_time and end_time be modified as well? Considering those are just the minutes past 0:00 that the availability starts and ends respectively. Example 540/60 is 9 (9 AM) and 1020/60 is 17 (5 PM), is this design fine as well?

Thanks!